src/models/users.js

```diff
@@ -0,0 +1,14 @@
+const { DataTypes } = require("sequelize");
+const sequelize = require("../utils/db.js");
+
+const Users = sequelize.define('Users', {

```

Comment:
you need to set up id for user like or use uiid type
![image](https://github.com/mate-academy/node_accounting-app-with-db/assets/96265844/6aa62b22-f6ce-4316-ac58-b5d22c6653f7)